src/inc/quic_platform_posix.h

```diff
@@ -559,21 +559,24 @@ typedef struct CXPLAT_EVENT_OBJECT {
     //
     // Mutex and condition.
     //
-
+    int Sentinel0;
     pthread_mutex_t Mutex;
     pthread_cond_t Cond;
 
     //
     // Denotes if the event object is in signaled state.
     //
 
+    char Sentinel1;
+
     BOOLEAN Signaled;
 
     //
     // Denotes if the event object should be auto reset after it's signaled.
     //
 
     BOOLEAN AutoReset;
+    char Sentinel2;
```

Comment:
Can these be removed? I assume they were there for testing?

Reply:
oops. yes. that should go.